CONTRIBUTING.md

```diff
@@ -51,12 +50,23 @@ and now we can run the local Jupyter Lab and run the scripts etc. using Poetry:
 
 - Create an issue to discuss the feature's scope and its fit for this package
 - run pytest to ensure your local version of code passes all unit tests
+```
+pytest tests/unit
+```
+
```

Comment:
There's a `Makefile`, so this should be `make test`.

Reply:
I suspect this may be a knock-on effect from your poetry problem mentioned elsewhere, such that `pytest-cov` didn't get installed.

Can you successfully run these commands, as given at the top of the guide (possibly with `conda` in place of `mamba`):

```plain
mamba env update -f ci/environment-dev.yml
mamba activate earthaccess-dev
poetry install
```